crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -1037,6 +1046,10 @@ pub struct SuiConsensusCommitPrologue {
     pub commit_timestamp_ms: u64,
 }
 
+#[serde_as]
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, PartialEq, Eq)]
+pub struct SuiAuthenticatorStateUpdate {}
```

Comment:
Looks like we made a choice not to expose data from this txn to jsonrpc - is this because clients can't do much with this data?
Even so, I would still expose the data just for uniformity with other RPCs, unless there is other reason to hide this data

Reply:
done